CHANGES.md

```diff
@@ -3,12 +3,21 @@ Change Log
 
 ### 1.11 - 2015-07-01
 
+* Deprecated
+  * Deprecated `AxisAlignedBoundingBox.intersect` and `BoundingSphere.intersect`.  These will be removed in 1.13.  Use `.intersectPlane` and, if necessary, `Plane.fromCartesian4`.
```

Comment:
Please submit an issue for this and another for the OOBB class (for example, see #2724).